### PR TITLE
#619 update gradle

### PR DESCRIPTION
**Pull request by onixred**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -69,6 +74,11 @@ dependencies {
     testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.10.1'
     testImplementation 'org.junit.jupiter:junit-jupiter-params:5.10.1'
     testImplementation 'org.junit.platform:junit-platform-engine:1.10.1'
+    modules {
+        module("com.google.collections:google-collections") {
+            replacedBy("com.google.guava:guava", "Google Collections is part of Guava")
+        }
+    }
```
Comment (McPringle) [inline build.gradle:81]: Please remove this module section and don't upgrade to checkstyle v10.12.6. The issue with the transitive dependency is on their side and will hopefully be fixed with v10.12.7. I reverted the change done by the renovate bot.

File: build.gradle [hunk 2]
```diff
@@ -23,17 +23,22 @@ plugins {
     id 'java-library'
     id 'org.bytedeco.gradle-javacpp-platform' version '1.5.9'
     id 'io.freefair.lombok' version '8.4'
+    id 'maven-publish'
```
Comment (McPringle) [inline build.gradle:26]: Please remove this plugin from this pull request. This PR is named "update gradle" and should not be mixed with other stuff. You are very welcome to add another PR with the Maven stuff!

File: build.gradle [hunk 3]
```diff
@@ -142,6 +152,15 @@ task createVersionFile {
...
 
+publishing {
+    publications {
+        myPublicationName(MavenPublication) {
+            // Configure the publication here
+            from components.java
+        }
+    }
+}
+
```
Comment (McPringle) [inline build.gradle:163]: Please remove this plugin from this pull request. This PR is named "update gradle" and should not be mixed with other stuff. You are very welcome to add another PR with the Maven stuff!

**Review by McPringle — CHANGES_REQUESTED** [review]
Thank you very much for your work on GPX Animator! I'll be very happy to merge this PR when you solve the two issues I mentioned inline.

File: build.gradle (hunk 1 above)
Comment (onixred) [reply build.gradle]: fixed

File: build.gradle (hunk 2 above)
Comment (onixred) [reply build.gradle]: fixed

File: build.gradle (hunk 3 above)
Comment (onixred) [reply build.gradle]: fixed

**Review by McPringle — APPROVED** [review]
Thank you very much, I'll merge this ASAP!

**Pull request merged**